### DISCUSSION TITLE
Analog clock on hover?

### DISCUSSION THREAD

**Discussion by ytdlder** [martinet101/ElevenClock#1050] [General]
I think it was Win7 that had that.
Would it be possible for you to add another hover option to show an analog clock?

That would be really useful imho :-)

Cheers!

**Comment by marticliment** [comment]
Honesly, this feature is kind of difficult, and it will be hard to ge it to show on the tooltip.

I'll take a look, but I can't guarantee anything...

**Reply by ytdlder** [reply]
Thanks for getting back to me so quickly.

Look, I've played around quite a bit more and... Explorer Patcher has the old Win7 clock included ;-)
I don't really wanna "patch" the system like that, but feature wise, I don't really have any option...

So, no worries!
